code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,448 @@
+		/datum/spacevine_mutation/toxicity,
+		/datum/spacevine_mutation/thorns,
+		/datum/spacevine_mutation/fire_proof,
+		/datum/spacevine_mutation/aggressive_spread,
+		)
+	// Items to pick from to spawn on examine
+	var/list/items_to_spawn = list(
+		/obj/item/food/grown/poppy,
+		/obj/item/food/grown/harebell,
+	)
+
+/datum/mood_event/eldritch_painting/heretic_vines
+	description = "Oh what a lovely flower!"
+	mood_change = 3
+	timeout = 1 MINUTES
+
+/obj/structure/sign/painting/eldritch/vines/Initialize(mapload, dir, building)
+	new /datum/spacevine_controller(get_turf(src), mutations, 0, 10)
+	return ..()
+
+/obj/structure/sign/painting/eldritch/vines/examine(mob/living/carbon/user)
+	..()
+	if("heretic_vines" in user.mob_mood.mood_events)
+		return
+
+	if(IS_HERETIC(user))
+		var/item_to_spawn = pick(items_to_spawn)
+		to_chat(user, span_notice("You picture yourself in the thicket picking flowers.."))
+		// Spawns one of two flowers that often are used by heretics
+		new item_to_spawn(user.drop_location())
+		user.add_mood_event("heretic_vines", /datum/mood_event/eldritch_painting/heretic_vines)
+	else
+		new /datum/spacevine_controller(get_turf(user), mutations, 0, 10)
+		to_chat(user, span_notice("The thicket crawls through the frame, and you suddenly find vines beneath you..."))
+
+
+
+// Lady out of gates, gives a brain trauma causing the person to scratch themselves
+/obj/item/wallframe/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her..."
+	icon_state = "eldritch_painting_beauty"
+	result_path = /obj/structure/sign/painting/eldritch/beauty
+
+/obj/structure/sign/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_beauty"
+	sensor_type = /datum/proximity_monitor/advanced/eldritch_painting/beauty
+	// Set to mutadone by default to remove mutations
+	var/list/reagents_to_add = list(/datum/reagent/medicine/mutadone)
+
+// The special examine interaction for this painting
+/obj/structure/sign/painting/eldritch/beauty/examine(mob/living/carbon/human/user)
+	..()
+	if(IS_HERETIC(user))
+		to_chat(user, "Your imperfections shed and you are restored.")
+		user.reagents.add_reagent(reagents_to_add, 5)
+	if(!user.has_dna())
+		return
+	if(user.has_trauma_type(/datum/brain_trauma/severe/eldritch_beauty))
+		to_chat(user, "You feel changed, more perfect....")
+		user.easy_random_mutate(NEGATIVE + MINOR_NEGATIVE)
+
+
+// Specific proximity monitor for Lady out of gates or /obj/item/wallframe/painting/eldritch/beauty
+/datum/proximity_monitor/advanced/eldritch_painting/beauty
+	applied_trauma = /datum/brain_trauma/severe/eldritch_beauty
+	text_to_display = "Her flesh glows in the pale light, and mine can too...If it wasnt for these imperfections...."
+
+/datum/brain_trauma/severe/eldritch_beauty
```

Comment:
similarly, why are these here?

Reply:
Ease of reading/following the code, I can move these

Reply:
Moved